mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/identifierdialog/IdentifierDialogFragment.java

```diff
@@ -52,6 +52,8 @@
     @Inject
     IdentifierDialogPresenter mIdentifierDialogPresenter;
 
+    ClientIdentifierCreationListener clientIdentifierCreationListener;
+
     View rootView;
```

Comment:
View should be ```private``` even if this is not your change, please do it since we've spotted the problem in this review.